src/upgrades/1.13.4/remove_allowFileUploads_priv.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const db = require('../../database');
+const privileges = require('../../privileges');
+
+module.exports = {
+	name: 'Removing file upload privilege if file uploads were disabled (`allowFileUploads`)',
+	timestamp: Date.UTC(2020, 4, 21),
+	method: async () => {
+		const allowFileUploads = parseInt(await db.getObjectField('config', 'allowFileUploads'), 10);
+		if (allowFileUploads === 1) {
+			return;
```

Comment:
Should probably call `await db.deleteObjectField('config', 'allowFileUploads');`  or the field will stay in db after upgrade

Reply:
I do this at the end of the file :thinking: unless I'm misunderstanding?

Reply:
oh, I am misunderstanding.